lib/auth/auth_test.go

```diff
@@ -115,6 +115,9 @@ func (s *AuthSuite) SetUpTest(c *C) {
 	err = s.a.SetAuthPreference(authPreference)
 	c.Assert(err, IsNil)
 
+	err = s.a.SetClusterAuditConfig(context.TODO(), types.DefaultClusterAuditConfig())
```

Comment:
define `ctx := context.Background()` at the top and replace `context.TODO()`s with `ctx`.